autonomy/templates/registration/login.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h3>Log In <small class="text-muted">to create your own tasks!</small></h3>
+    <hr>
+    {% if form.errors %}
+        <p>Your username and password didn't match. Please try again.</p>
+    {% endif %}
+
+    {% if next %}
+        {% if user.is_authenticated %}
+            <p>Your account doesn't have access to this page. To proceed, please login with an account that has access.</p>
+        {% else %}
+            <p>Please login to see this page.</p>
+        {% endif %}
+    {% endif %}
+
+    <form method="post" action="{% url 'login' %}" class="mx-auto">
+        {% csrf_token %}
+        <label for="id_username">Username</label>
+        <input class="form-control" type="text" name="username" id="id_username" required>
+        <label for="id_password">Password</label>
+        <input class="form-control" type="password" name="password" id="id_password" required>
+        <input type="submit" value="login" class="btn btn-success">
```

Comment:
I dont know if this matters too much, but we havent imported bootstrap in base.html, we should ask Donny if we should. If not lets remove the bootstrap css stuff

Reply:
Donny gave us the go ahead on bootstrap! (now we wont feel so incompetent :D)